pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/certmanager/certificate.go

```diff
@@ -49,6 +49,9 @@ const certManagerTemplate = `# The following manifests contain a self-signed iss
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
+  labels:
+    app.kubernetes.io/name: selfsigned-issuer
+    app.kubernetes.io/component: certificate
   name: selfsigned-issuer
```

Comment:
WDYT?
app.kubernetes.io/part-of -> cert-manager.io
app.kubernetes.io/managed-by -> kustomize 
app.kubernetes.io/created-by | {{Project Name}}
app.kubernetes.io/instance > the name of the resource 
 app.kubernetes.io/version -> v1